core/src/main/java/hudson/slaves/CloudRetentionStrategy.java

```diff
@@ -47,6 +47,20 @@ public CloudRetentionStrategy(int idleMinutes) {
         this.idleMinutes = idleMinutes;
     }
 
+    /**
+     * @since FIXME
+     */
+    public int getIdleMinutes() {
+        return idleMinutes;
+    }
+
+    /**
+     * @since FIXME
+     */
+    public void setIdleMinutes(int idleMinutes) {
```

Comment:
Is setter really required? This field is rather a final one according to the current usage. I would vouch against setter unless there is a use-case for it